dev/src/lobster/glinterface.h

```diff
@@ -207,6 +207,11 @@ struct Mesh : lobster::Resource {
     float3x4 *mats = nullptr;
     float curanim = 0;
 
+    struct Animation {
+        uint32_t first_frame, num_frames;
```

Comment:
Just use `int`.. all Lobster values are signed, so less unsigned values means less conversions on the C++ side.